modules/govuk/manifests/node/s_postgresql_standby.pp

```diff
@@ -10,8 +10,21 @@
 #
 class govuk::node::s_postgresql_standby (
   $primary_password,
+  $aws_access_key_id,
+  $aws_secret_access_key,
+  $s3_bucket_url,
+  $wale_private_gpg_key,
```

Comment:
I think we should name these `wal_e` for consistency with the Puppet class.

Reply:
They'd be consistent with the Puppet class, but inconsistent with the `s_postgresql_primary` class.  Unless that were to change too, which would require changing secrets.

Reply:
Hmm yeah, that's unfortunate. Probably better in this case to be consistent with the `s_postgresql_primary` class.